plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/SnapshotsTable.java

```diff
@@ -28,20 +28,34 @@
 import io.trino.spi.type.TimeZoneKey;
 import io.trino.spi.type.TypeManager;
 import io.trino.spi.type.TypeSignature;
+import org.apache.iceberg.DataTask;
+import org.apache.iceberg.StructLike;
 import org.apache.iceberg.Table;
+import org.apache.iceberg.TableScan;
 
 import java.util.List;
+import java.util.Map;
 
+import static io.trino.plugin.iceberg.IcebergUtil.buildTableScan;
+import static io.trino.plugin.iceberg.IcebergUtil.columnNameToPositionInSchema;
 import static io.trino.spi.type.BigintType.BIGINT;
 import static io.trino.spi.type.TimestampWithTimeZoneType.TIMESTAMP_TZ_MILLIS;
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static java.util.Objects.requireNonNull;
+import static org.apache.iceberg.MetadataTableType.SNAPSHOTS;
 
 public class SnapshotsTable
         implements SystemTable
 {
     private final ConnectorTableMetadata tableMetadata;
     private final Table icebergTable;
+    private static final String committedAtColumnName = "committed_at";
+    private static final String snapshotIdColumnName = "snapshot_id";
+    private static final String parentIdColumnName = "parent_id";
+    private static final String operationColumnName = "operation";
+    private static final String manifestListColumnName = "manifest_list";
+    private static final String summaryColumnName = "summary";
+    private static final long MICROSECONDS_PER_MILLISECOND = 1000L;
```

Comment:
Static final constants should use names that are in ALL_CAPS

`MICROSECONDS_PER_MILLISECOND` is already defined, it can be statically imported from 
`import static io.trino.spi.type.Timestamps.MICROSECONDS_PER_MILLISECOND;`